### DISCUSSION TITLE
Disabling auto onboarding for our organisation

### DISCUSSION THREAD

**Discussion by bjw-s** [renovatebot/renovate#18432] [Request Help]
Hi, we would like to enable Renovate for all repo's in our organisation (http://github.com/sectigo), but this would lead to a lot of unwanted onboarding PR's being created.

I noticed in https://github.com/renovatebot/renovate/discussions/17134 and similar topics that we should request a modification here.

Could you please set the `"onboarding":false` setting for the Sectigo GitHub organisation so that Renovate will run only on those repo's with a pushed configuration but without creating PR's for the ones that don't have a config?

Thank you very much for your assistance!

**Comment by rarkins** [comment]
I have done this. Please double check first by adding an additional repo to confirm it's working as expected, before you install into all

**Reply by bjw-s** [reply]
I've just verified with a test repo and all looks well. Thanks for the help!
